src/realm-settings/KeysProvidersTab.tsx

```diff
@@ -317,22 +349,24 @@ export const KeysTabInner = ({ components, refresh }: KeysTabInnerProps) => {
           {(filteredComponents.length === 0
             ? components
             : filteredComponents
-          ).map((component: ComponentData, idx) => (
+          ).map((component: ComponentData, idx: number) => (
```

Comment:
The types can be inferred here from the value of the array we're mapping on, so the type annotations can be omitted.
```suggestion
          ).map((component, idx) => (
```